visualizer/ui.js

```diff
@@ -331,6 +332,20 @@ class Ui extends events.EventEmitter {
     return keysToLabels[key] || key
   }
 
+  getDescriptionFromKey (key) {
+    const keysToDescriptions = {
+      'core': `JS functions in core Node.js APIs. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#core">More info</a>`,
+      'all-v8': 'The V8 engine JavaScript implementation',
+      'v8': `Operations in V8's implementation of JS. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#v8">More info</a>`,
+      'native': `JS compiled into V8, such as prototype methods and eval. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#native">More info</a>`,
+      'cpp': `Native C++ operations called by V8, including shared libraries. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#cpp">More info</a>`,
+      'regexp': `The RegExp notation is shown as the function name. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#rx">More info</a>`,
+      'init': `Module loading and other operations during initialization. Frames in this category also belong to other categories. <a target="_blank" class="more-info" href="https://clinicjs.org/flame/walkthrough/controls/#init">More info</a>`,
```

Comment:
This is a bit long, and it doesn't feel like a great fit here. How about we move it to the top of the second heading section, above merged, where it'll be more intuitive that it is something that crosses other sections, and call it:

> **Init** - Show initialization operations hidden by default, like module loading.

... and then rename that section to something like "Advanced"

Reply:
My first thought was to add a different section but I didn't want to do that for just this item. Changing the existing one to an "advanced" section sounds good!